keyboards/foxlab/leaf60/hotswap/rules.mk

```
@@ -69,7 +69,7 @@ COMMAND_ENABLE = no        # Commands for debug and configuration
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-NKRO_ENABLE = no            # USB Nkey Rollover
+NKRO_ENABLE = yes            # USB Nkey Rollover
```

Please revert this: 
```suggestion
NKRO_ENABLE = no            # USB Nkey Rollover
```

If this is needed, then please feel free to move this to the rules.mk for the keymap, not for the whole keyboard.